2023-07-29 12:32:00,030.continuous_location.INFO =============================================================
2023-07-29 12:32:00,030.continuous_location.INFO Today is 2023-07-29 at 12:32 PM
2023-07-29 12:32:00,036.continuous_location.INFO This file is running on: posix Darwin 19.6.0
2023-07-29 12:32:00,036.continuous_location.INFO The Python version is: 3.8.8
2023-07-29 12:32:00,036.continuous_location.INFO The active environment path is:   /Users/ashleymersman/Desktop/Continuous_Intelligence/cintel-05-live-updates/.venv
2023-07-29 12:32:00,037.continuous_location.INFO The current working directory is: /Users/ashleymersman/Desktop/Continuous_Intelligence/cintel-05-live-updates
2023-07-29 12:32:00,037.continuous_location.INFO =============================================================
2023-07-29 12:32:16,323.continuous_location.INFO Calling update_csv_location
2023-07-29 12:32:16,323.continuous_location.INFO update_interval: 60
2023-07-29 12:32:16,323.continuous_location.INFO total_runtime: 900
2023-07-29 12:32:16,323.continuous_location.INFO num_updates: 10
2023-07-29 12:32:16,324.continuous_location.INFO Initialized csv file at /Users/ashleymersman/Desktop/Continuous_Intelligence/cintel-05-live-updates/data/mtcars_location.csv
2023-07-29 12:32:16,324.continuous_location.INFO Calling get_temperature_from_openweathermap for {lat}, {long}}
2023-07-29 12:32:16,325.continuous_location.INFO Calling fetch_from_url for https://api.openweathermap.org/data/2.5/weather?lat=47.903237&lon=-91.867087&appid=None&units=imperial
2023-07-29 12:32:16,598.continuous_location.ERROR ERROR in update_csv_location: HTTP Error 401: Unauthorized
